src/translations/en.json

```diff
@@ -1844,6 +1844,7 @@
         "component_not_loaded": "This redirect is not supported by your Home Assistant instance. You need the integration {integration} to use this redirect.",
         "no_supervisor": "This redirect is not supported by your Home Assistant installation. It needs either the Home Assistant Operating System or Home Assistant Supervised installation method. For more information, see the {docs_link}.",
         "not_app": "This redirect only works from a mobile device that has the Home Assistant Companion app installed. {link}.",
+        "url_error": "This url had some unexpected characters or formatting that did not match the sanitized version.",
```

Comment:
The message here is too specific.  This will be shown for any invalid URL, so it should be more generic.